examples/forward/left_cerebellum_volume_source.py

```diff
@@ -73,8 +73,8 @@
 # And display source positions in freeview::
 #
 #    >>> from mne.utils import run_subprocess
-#    >>> mri_fname = subjects_dir + '/sample/mri/brain.mgz'
-#    >>> run_subprocess(['freeview', '-v', mri_fname, '-v',
-#                        '%s:colormap=lut:opacity=0.5' % aseg_fname, '-v',
-#                        '%s:colormap=jet:colorscale=0,2' % nii_fname,
-#                        '-slice', '157 75 105'])
+#    >>> mri_fname = subjects_dir / "sample" / "mri" / "brain.mgz"
+#    >>> run_subprocess(["freeview", "-v", str(mri_fname), "-v",
+#                        "%s:colormap=lut:opacity=0.5" % str(aseg_fname),
```

Comment:
Could replace the %s with f-strings too?

Reply:
done